apache-rat-plugin/src/main/java/org/apache/rat/mp/AbstractRatMojo.java

```diff
@@ -78,7 +78,7 @@ public abstract class AbstractRatMojo extends BaseRatMojo {
      * Specifies the licenses to accept. By default, these are added to the default
      * licenses, unless you set &lt;addDefaultLicenseMatchers&gt; to false. Arguments should be
      * file name of &lt;Configs&gt; file structure.
-     * @deprecated Use specific configuration under &lt;configuration&gt;.
+     * @deprecated Use &lt;config&gt;.
```

Comment:
I introduced these changes in other pullRequests as there is no config-element at all - would it make sense to merge in master into this branch? as it reverts those changes forcefully @Claudenw